ui/src/components/yaml/index.tsx

```diff
@@ -377,35 +380,45 @@
                   />
                 </Space>
```

Comment:
The `ref={contentRef}` has been added to the new div. This might be for scrolling or positioning purposes.